.github/workflows/test.yml

```diff
@@ -11,9 +11,8 @@ env:
 
 jobs:
   build:
-
+    name: tests
```

Comment:
Not going to bikeshed this to death but "hubris-tests" might be more specific. Up to you if you think we're likely to grow other tests.

Reply:
Eh, I think this is fine for now since all the other CI steps are also short (`dist`, `formatting`, `documentation`)